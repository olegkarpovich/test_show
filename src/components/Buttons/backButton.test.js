import React from 'react';
import { render, act } from '@testing-library/react';
import { BackButton } from './index';

const component = () => (
    <BackButton />
);

describe('BackButton component', () => {
    it('should render back button', () => {
        const { container } = render(component());

        expect(container).toMatchSnapshot();
    });
});