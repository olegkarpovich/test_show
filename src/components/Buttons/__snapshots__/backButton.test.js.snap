// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackButton component should render back button 1`] = `
<div>
  <button
    id="back-to-show-button"
  >
    Back
  </button>
</div>
`;
